ws-nextjs-app/pages/[service]/av-embeds/AvEmbedsMetadata.test.tsx

```diff
@@ -130,10 +129,6 @@ describe('AV Embeds Page', () => {
       },
       { content: 'BBC News', property: 'og:site_name' },
       { content: 'sr-Cyrl', property: 'og:locale' },
-      {
-        content: 'https://www.bbc.com/serbian/cyr/av-embeds/srbija-68707945',
-        property: 'og:url',
-      },
```

Comment:
Is this expected?

Reply:
Yes this will not be set by this component any more, but rather the new Metadata component: https://github.com/bbc/simorgh/blob/b4c30dca6315d6dc812b3f5e11884a12501e2a4a/src/app/components/MediaLoader/Metadata/index.tsx#L83

Reply:
I'll check the test and see if I can check its existence in that test suite.

Reply:
Thanks!